tempus_fugit_minecraft/player.py

```diff
@@ -97,15 +98,25 @@ def get_motion_vector(self) -> tuple:
             dx = 0.0
             dz = 0.0
         return dx, dy, dz
-    
-    def speed_up(self) -> None:
-        """Increases the walking speed of the player."""
+ 
+    """Issue #71 """
+    def increase_speed(self) -> None:
         if self.walking_speed <= 15:
             self.walking_speed += self.WALK_SPEED_INCREMENT
-
-    def speed_down(self):
+ 
+    """Issue #71 """   
+    def decrease_speed(self):
         if self.walking_speed > 5:
             self.walking_speed -= self.WALK_SPEED_INCREMENT
+   
+    #Issue #39
+    def increase_jump(self) -> None:       
+            if self.jump_speed <= 15: 
+                self.jump_speed += 5
+    #Issue #39
+    def decrease_jump(self) -> None:
+        if self.jump_speed >= 10:       
+            self.jump_speed -= 5
```

Comment:
Feels like these functions should be relative to the current walking speed rather than the jump_speed. If you want to limit the jump_speed, I think a hard cap would be added, such as max_jump_speed, min_jump_speed.

Reply:
Good one, I added 2 variables for max and min, Thank you

Reply:
Updated as bool